crates/crabe_framework/src/data/geometry.rs

```diff
@@ -46,30 +46,47 @@ impl Default for Geometry {
                 length: 9.0,
                 width: 6.0,
             },
-            ally_goal: Goal {
-                width: 1.0,
-                depth: 0.18,
-                top_left_position: Point2::new(-4.68, -0.5),
-            },
-            enemy_goal: Goal {
-                width: 1.0,
-                depth: 0.18,
-                top_left_position: Point2::new(4.68, 0.5),
-            },
+
+            // You might ask "Why is goal width not the rectangle's width here ???"
+            // To this I answer "No fucking idea"
```

Comment:
I understand there may be frustration, but it's important for us to maintain a professional tone in the comments and stay focused on the project objectives. 
This comment isn't constructive or relevant to our goals.
Let's aim to keep our feedback helpful and directly related to the task at hand.

Reply:
To answer your question about `goal_length` and `goal_width`.
`goal_length` doesn't existe.

![image](https://github.com/NAMeC-SSL/CRAbE/assets/20494573/5fc91830-23a5-454e-b256-2ed87e72deef)

Reply:
alright i'll change this to something more pro

i didn't quite catch what you said in the second post though, what I mentioned is already what I implemented. Should I change something ?

Reply:
as we discussed, this is what we wanted already, so I will edit this PR to change the struct accordingly